client/src/charts/legends/SelectAllControl.js

```diff
@@ -3,10 +3,13 @@ import { TrivialTM } from "../../components/index.js";
 export const SelectAllControl = ({ SelectAllOnClick, SelectNoneOnClick }) => (
   <div
     role="group"
-    aria-labelledby={"SelectAllControlGroup"}
+    aria-label={"SelectAllControlGroup"}
```

Comment:
Screenreader now says "SelectAllControlGroup", not sure we need this? inside the group it already says "select", "all", and "none"

Reply:
yeah I actually don't even think we need aria label here